### ISSUE TITLE
Aria-label for "Main navigation" should be "Main"

### ISSUE THREAD

**Issue by IsabelBirds** [executablebooks/sphinx-book-theme#308]
**Describe the bug**

When screen readers read a "nav" element they already say "Navigation", this currently reads "Main Navigation navigation". This is a very small issue, but will improve the experience for anyone using a screenreader.

https://github.com/executablebooks/sphinx-book-theme/blob/master/sphinx_book_theme/_templates/sbt-sidebar-nav.html

**To Reproduce**

1. Using VoiceOver screenreader access Rotor (control-option-U)
2. Use left/right arrow keys to navigation to "Landmarks"
3. You should be able to see Main navigation navigation element

**Expected behavior**
Removing "navigation" from the aria label will mean it will read "Main, navigation".

**Environment**

- Operating System: MacOS Voiceover 

**Additional context**

**Comment by choldgraf** [comment]
Thanks for bringing up this accessibility issue, I think these sort of things are super important. I believe that the change would be somewhere around here

https://github.com/executablebooks/sphinx-book-theme/blob/bdcf1065f0fbf92f786082b78c86af7a280651df/sphinx_book_theme/layout.html#L27

however I can’t figure out where the aria label is being set, perhaps it is something that we are inheriting from the pydata theme?
